utils/include/CJSONFormatSaver.cpp

```diff
@@ -4,31 +4,67 @@
 #include "CJSONFormatter.h"
 #include "CJSONFormatSaver.h"
 
+CJSONFormatSaver::CJSONFormatSaver(const std::string& path_to_file) :
+	m_path_to_file(path_to_file), m_number_of_spaces(3)
+{ };
+
 bool CJSONFormatSaver::TrySaveToFile(CJSONFormatter& formatted_data)
```

Comment:
Maybe formatted_data should be const ?

Reply:
I have changed all places where was used this method, according to new changes (made meth as const)